src/main/java/york/pharmacy/orders/Order.java

```diff
@@ -0,0 +1,46 @@
+package york.pharmacy.orders;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
```

Comment:
The doc for `AuditingEntityListener` says: 

> JPA entity listener to capture auditing information on persisting and updating entities. To get this one flying be sure you configure it as entity listener in your orm.xml as follows:
> ```
> <persistence-unit-metadata>
>      <persistence-unit-defaults>
>          <entity-listeners>
>              <entity-listener class="org.springframework.data.jpa.domain.support.AuditingEntityListener" />
>          </entity-listeners>
>      </persistence-unit-defaults>
>  </persistence-unit-metadata>
> ```

Do we need an `orm.xml` file?